scripts/makeUMAPplots.R

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env Rscript
+## script to create two UMAP plots for each reference to allow quality control
+## of reference and reduced cell type labels
+suppressMessages(library(Seurat))
+suppressMessages(library(cowplot))
+suppressMessages(library(ggplot2))
+
+args = commandArgs(trailingOnly=TRUE)
+filename <- args[1]
+output_file_name <- args[2]
+
+# Read in the Seurat object and store it 
+seurat_obj <- readRDS(filename)
+# run basic steps in order to gerneat UMAP plot from SeuratObject
+seurat_obj <- FindVariableFeatures(seurat_obj, selection.method = "vst", nfeatures = 2000)
+seurat_obj <- ScaleData(seurat_obj)
+seurat_obj <- RunPCA(seurat_obj, features = VariableFeatures(object = seurat_obj))
+seurat_obj <- RunUMAP(seurat_obj, dims = 1:10)
+Idents(seurat_obj) = seurat_obj$cell_type_names
+#plot old cell type labels before running reduceCellTypes.R
+p1 = DimPlot(seurat_obj, 
```

Comment:
Please use `<-` instead of `=` in the R scripts
```suggestion
p1 <- DimPlot(seurat_obj, 
```